interface/forms/prior_auth/templates/prior_auth/general_new.html

```diff
@@ -34,20 +48,134 @@
 
 -->
 </style>
+<script>
+function showHint(str) {
+    if (str.length == 0) { 
+        document.getElementById("txtHint").innerHTML = "";
+        return;
+    } else {
+        var xmlhttp = new XMLHttpRequest();
+        xmlhttp.onreadystatechange = function() {
+            if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+                document.getElementById("txtHint").innerHTML = xmlhttp.responseText;
+            }
+        };
+        xmlhttp.open("GET", "gethint.php?q=" + str, true);
+        xmlhttp.send();
+    }
+}
+
+ function validate() {
+            var EnteredDate = document.getElementById("auth_from").value; // get the auth from date
+
+            var today = document.getElementById("auth_to").value;
+
+            if (EnteredDate > today) {
+                alert("FROM: date is greater than TO: date ");
```

Comment:
translate and escape with xls function:
``````
alert("<?php echo xls('The FROM date is greater than the TO date'); ?>");